tests/src/main/kotlin/se/ansman/kotshi/ClassWithJavaKeyword.kt

```diff
@@ -0,0 +1,17 @@
+package se.ansman.kotshi
+
+import com.squareup.moshi.Json
+
+@JsonSerializable
+data class ClassWithJavaKeyword(
+        @GetterName("getDefault")
+        @Json(name = "default")
+        val default: Boolean,
+        @GetterName("getInt")
+        @Json(name = "int")
+        val int: Int,
+        @GetterName("someCase")
+        @Json(name = "case")
+        @get:JvmName("someCase")
```

Comment:
We should figure out how to read the `@Metadata` annotation in an annotation processor so we can avoid all this ceremony. That's been long on my todo list...

Reply:
Yeah, that would be fantastic. I looked in to that previously to try and figure out if I could use default values that way but it was too much work at the time but now that the project is stable it would be a fun project